modules/puppet_proxy/configuration_loader.rb

```diff
@@ -1,8 +1,7 @@
 module ::Proxy::Puppet
   class ConfigurationLoader
     def load_programmable_settings(settings)
-      use_provider = settings[:use_provider]
-      use_provider = [use_provider].compact unless use_provider.is_a?(Array)
+      use_provider = []
       use_provider << :puppet_proxy_puppet_api
       settings[:use_provider] = use_provider
```

Comment:
Should we simplify this:
```suggestion
      settings[:use_provider] = [:puppet_proxy_puppet_api]
```

Reply:
yes. make sense, thanks @ekohl